cortex-js/src/infrastructure/commanders/models/model-start.command.ts

```diff
@@ -9,16 +14,22 @@ type ModelStartOptions = {
 @SubCommand({ name: 'start', description: 'Start a model by ID.' })
 export class ModelStartCommand extends CommandRunner {
   constructor(
+    private readonly inquirerService: InquirerService,
     private readonly cortexUsecases: CortexUsecases,
     private readonly modelsCliUsecases: ModelsCliUsecases,
   ) {
     super();
   }
 
   async run(input: string[], options: ModelStartOptions): Promise<void> {
-    if (input.length === 0) {
-      console.error('Model ID is required');
-      exit(1);
+    let modelId = input[0];
+    if (!modelId) {
+      try {
+        modelId = await this.modelInquiry();
+      } catch {
```

Comment:
maybe just log the error?

Reply:
No, can't log error here. 

This is a fallback for the case modelId is not inputted. Not a happy case to log weird error out.